index.html

```diff
@@ -2307,6 +2991,350 @@ <h2>
         </table>
       </section>
     </section>
+    <section>
+      <h2>
+        Shipping and billing addresses
+      </h2>
+      <p>
+        The {{PaymentRequest}} interface allows a merchant to request from the
+        user [=physical address|physical addresses=] for the purposes of
+        shipping and/or billing.
+      </p>
+      <section data-dfn-for="AddressInit">
```

Comment:
NOTE: we need to move all this to the Contact Picker API... along with creating an address from user input.

Reply:
So, yes, all this can go away....